shared/src/main/scala/com/raquo/domtypes/generic/defs/eventProps/FormEventProps.scala

```diff
@@ -19,7 +22,7 @@ import com.raquo.domtypes.generic.builders.EventPropBuilder
   *            DOM InputEvent https://developer.mozilla.org/en-US/docs/Web/API/InputEvent
   *            Note: This type is not implemented in scala-js-dom
   */
-trait FormEventProps[EP[_ <: DomEvent], DomEvent, DomFocusEvent <: DomEvent, DomInputEvent <: DomEvent, DomHtmlElementTargetEvent <: DomEvent, DomFormElementTargetEvent <: DomEvent, DomInputElementTargetEvent <: DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
+trait FormEventProps[EP[_ <: DomEvent], DomEvent, DomFocusEvent <: DomEvent, DomInputEvent <: DomEvent, DomElementTargetEvent <: DomEvent, DomHtmlElementTargetEvent <: DomEvent, DomFormElementTargetEvent <: DomEvent, DomInputElementTargetEvent <: DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
```

Comment:
I am not completely sure about the `target` types anymore. Why does [MDN](https://developer.mozilla.org/en-US/docs/Web/Events/submit) often say `target: Element`, but then in the description gives a more concrete type on which elements the event can occur?

Reply:
Well, conventional JS ideology is largely "let's try to run this and see what we get", so I don't imagine JS devs care all that much about what types are specified. There are also potentially some differences in what is specified in the DOM standard and what is actually implemented in browsers, but I'd think those are minimal by now. I think it's mostly just that JS users largely don't care about minor typing details like this.

For the `submit` event MDN says "Note that submit is fired only on the form element, not the button or submit input. (Forms are submitted, not buttons.)", and I've verified this behavior manually in a couple browsers, so I'm confident enough to put `Form` as the type of target for such an event.